Xamarin.Forms.Platform.Android/CollectionView/ItemsViewAdapter.cs

```diff
@@ -1,23 +1,22 @@
 using System;
-using System.Diagnostics;
 using Android.Content;
 using Android.Support.V7.Widget;
-using Android.Views;
 using Android.Widget;
 using AView = Android.Views.View;
 using ViewGroup = Android.Views.ViewGroup;
 
 namespace Xamarin.Forms.Platform.Android
 {
-	// TODO hartez 2018/07/25 14:39:29 Split up CollectionViewAdapter into one for templates, one for text	
-	// TODO hartez 2018/07/25 14:43:04 Experiment with an ItemSource property change as _adapter.notifyDataSetChanged	
+	// TODO hartez 2018/07/25 14:39:29 Split up CollectionViewAdapter into one for templates, one for text
+	// TODO hartez 2018/07/25 14:43:04 Experiment with an ItemSource property change as _adapter.notifyDataSetChanged
 	// TODO hartez 2018/07/25 14:44:15 Template property changed should do a whole new adapter; and that way we can cache the template
 
 	public class ItemsViewAdapter : RecyclerView.Adapter
 	{
 		protected readonly ItemsView ItemsView;
-		readonly Func<IVisualElementRenderer, Context, AView> _createView;
-		readonly IItemsViewSource _itemsSource;
+		Func<IVisualElementRenderer, Context, AView> _createView;
```

Comment:
Since these don't need to be modified (see comments below), the `readonly` can be reinstated.